### PR TITLE
adds a simple PR template

### PR DESCRIPTION
**Pull request by mitchell852**
This template will be applied when any new PRs are created against Traffic Control. The idea is that it will help capture more information for potential reviewers/mergers and lead to better PRs and faster merges.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2251/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2252/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2253/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2254/
Test PASSed.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#### What is the best way to verify this PR?
+
+
+#### Check any that apply
+
+- [ ] This PR does *NOT* fix a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)
+- [ ] This PR includes tests
+- [ ] This PR includes documentation updates
+- [ ] This PR includes an update to CHANGELOG.md
+
```
Comment (dangogh) [inline PULL_REQUEST_TEMPLATE.md]: does it need to mention newly introduced files that require a license header (like this one?) :-)

Comment (mitchell852) [reply PULL_REQUEST_TEMPLATE.md]: @dangogh - i was thinking that as well. how about another checkbox like:

- [ ] This PR includes all required license headers

Comment (mitchell852) [reply PULL_REQUEST_TEMPLATE.md]: I added the license header and a bullet for that

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2255/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2256/
Test FAILed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2258/
Test PASSed.

**Pull request merged**